src/benchmarklib/benchmark_utils.hpp

```diff
@@ -82,6 +82,11 @@ struct EncodingConfig {
   static const char* description;
 };
 
+class BenchmarkTableEncoder {
```

Comment:
Ignore me, this is just my usual rant about creating classes for everything. In this case you just want a function to encode a table, nice and clean.